app/Platform/Models/Game.php

```diff
@@ -65,7 +65,6 @@ class Game extends BaseModel implements HasComments, HasMedia
     public const UPDATED_AT = 'Updated';
 
     protected $fillable = [
-        'system_id',
```

Comment:
This is not needed?

Reply:
It doesn't exist. The field is "ConsoleID"

Reply:
good point! i think it was used for the seeders/test helpers but with factories that's apparently not required anymore. interesting